src/Components/RoomForm/RoomForm.js

```diff
@@ -42,43 +49,49 @@ function RoomForm() {
 
 
     return (
-      <section className="RoomForm">
-        <form onSubmit={ (e) => {e.preventDefault()} }>
-          <label htmlFor='room-input'>Roomname:
-            <input
-              id='room-input'
-              type='text'
-              placeholder='Name'
-              value={ roomname }
-              onChange={(e) => { setRoomname(e.target.value) }}
-            />
-          </label>
-          <h5>Room Code: { roomID }</h5>
-          <section>
-            {createRules()}
-          </section>
-          <label>
-            <input id='rule-input'
-              type='text'
-              placeholder='Rule'
-              value={ potentialRule }
-              onChange={ (e) => { setRule(e.target.value) } }
-              onKeyUp={ (e) => { handleEnter(e) } }
-            />
-          <button type='button' onClick={(e) => { addRule(e) }}>+</button>
+      <form onSubmit={ (e) => {e.preventDefault()} } className="RoomForm">
+        <section className='input-container'>
+          <label htmlFor='room-input'>Group:
           </label>
+          <input
+          id='room-input'
+          type='text'
+          placeholder='Name'
+          value={ roomname }
+          onChange={(e) => { setRoomname(e.target.value) }}
+          />
+        </section>
+        <section className='input-container'>
+          <label>Group Code:</label> <h2>{ roomID }</h2>
+        </section>
+        <h3 className='rules-head'>Group Rules</h3>
+        <section className='rules-container'>
+          {createRules()}
+          <div className='empty-space' ref={messagesEndRef} />
+        </section>
+        <section className='rules-input'>
+          <input id='rule-input'
+          type='text'
+          placeholder='Rule'
+          value={ potentialRule }
+          onChange={ (e) => { setRule(e.target.value) } }
+          onKeyUp={ (e) => { handleEnter(e) } }
```

Comment:
is this where we put the "press enter to add rule" functionality?

Reply:
yes handle enter checks that the enter key was the target and handles it the same as clicking the +button would do